examples/dmd/wave_equation.cpp

```diff
@@ -448,9 +501,17 @@ int main(int argc, char *argv[])
     fom_timer.Stop();
     dmd_training_timer.Start();
     int curr_window = 0;
+
     vector<CAROM::DMD*> dmd_u;
-    dmd_u.push_back(new CAROM::DMD(u.Size(), dt));
-    dmd_u[curr_window]->takeSample(u.GetData(), t);
+    if(snapshotDMD)
+    {
+        dmd_u.push_back(new CAROM::SnapshotDMD(u.Size(), dt));
+    }
+    else
+    {
+        dmd_u.push_back(new CAROM::DMD(u.Size(), dt));
+    }
+    dmd_u[curr_window]->takeSample(u, t);
```

Comment:
`u.GetData()` (to avoid deprecated warning)

Reply:
fixed